streams_chunks_test.go

```diff
@@ -0,0 +1,412 @@
+package lksdk
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+	"unicode/utf8"
+)
+
+func TestChunkUtf8String(t *testing.T) {
+	t.Run("Empty string", func(t *testing.T) {
+		chunks := chunkUtf8String("")
+		if len(chunks) != 0 {
+			t.Errorf("Expected 0 chunks for empty string, got %d", len(chunks))
```

Comment:
Does this fail the test suite? We use `require` package for testing. So, this would be something like `require.Equal(t, 0, len(chunks))` (maybe `require` has a convenience method called `require.Zero` as they have a lot of convenience methods). Would be good to stay consistent in testing.

Reply:
Yeah it fails the test, but let me update these to be consistent with other tests

Reply:
Done! [ea551c5](https://github.com/livekit/server-sdk-go/pull/622/commits/ea551c5ae0aba0f35a1fbc5d920c74f376f8334b)